Kavita.Common/Configuration.cs

```diff
@@ -200,10 +206,61 @@ private static string GetIpAddresses(string filePath)
     }
     #endregion
 
+    #region BaseUrl
+    private static string GetBaseUrl(string filePath)
+    {
+        if (new OsInfo(Array.Empty<IOsVersionAdapter>()).IsDocker)
+        {
+            return "/";
+        }
+
+        try
+        {
+            var json = File.ReadAllText(filePath);
+            var jsonObj = JsonSerializer.Deserialize<dynamic>(json);
+            const string key = "BaseUrl";
+
+            if (jsonObj.TryGetProperty(key, out JsonElement tokenElement))
+            {
+                var baseUrl = tokenElement.GetString();
+                return String.IsNullOrEmpty(baseUrl) ? "/" : baseUrl;
+            }
+        }
+        catch (Exception ex)
+        {
+            Console.WriteLine("Error reading app settings: " + ex.Message);
+        }
+
+        return "/";
```

Comment:
"/" is hardcoded in many places, extract to a Default variable, like done with IpAddresses. This will allow us to provide a reset, like we did with IpAdresses.